src/Display/Homepage.js

```diff
@@ -16,8 +16,7 @@ const Shows = () => {
       shouldGetPosts.current = false;
       dispatch(getPosts());
     }
-    // eslint-disable-next-line
-    },[]);
+  }, [dispatch]);
 
   return (
```

Comment:
- [ ] As suggested by the previous reviewer that you should have a way to filter your data such as you can create a search box on your homepage to filter(search) movies :blush: